tests/images/testimages.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+from typing import List
+
+from builder import (
+    ArtifactManifest,
+    BaseImage,
+    BaseImageManifest,
+    ImageConfig,
+    OutputFormat,
+    SystemArchitecture,
+    cli,
+)
+
+# # # # # # # # # # # # # # # # # # #
+#    DEFAULT CUSTOMIZER VERSION     #
+#                                   #
+# The version of Image Customizer   #
+# to use by default when building   #
+# images.                           #
+# # # # # # # # # # # # # # # # # # #
+DEFAULT_IMAGE_CUSTOMIZER_VERSION = "0.19"
+
+
+DEFINED_IMAGES: List[ImageConfig] = [
+    ImageConfig(
```

Comment:
should this list be slimmed down?  or are more config files going to be copied over to this PR?